### ISSUE TITLE
Warn about recipe which are pulling source from different domains

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#90]
Most likely that results in a mixup of licensing a.s.o.

e.g. should be warned about

```
SRC_URI = "git://some.corp.domain/main.git"
SRC_URI += "git://some.other.domain/foo.git"
```

while, of course, the following is valid

```
SRC_URI = "git://some.corp.domain/main.git"
SRC_URI += "file://extra.patch"
```
